README.md

```diff
@@ -79,4 +81,4 @@ If you are looking to make a code or documentation contribution to this project
 # Related projects
 
 ### Zigpy
-[zigpy](https://github.com/zigpy/zigpy)** is **[Zigbee protocol stack](https://en.wikipedia.org/wiki/Zigbee)** integration project to implement the **[Zigbee Home Automation](https://www.zigbee.org/)** standard as a Python 3 library. Zigbee Home Automation integration with zigpy allows you to connect one of many off-the-shelf Zigbee adapters using one of the available Zigbee radio library modules compatible with zigpy to control Zigbee based devices. There is currently support for controlling Zigbee device types such as binary sensors (e.g., motion and door sensors), sensors (e.g., temperature sensors), lightbulbs, switches, and fans. A working implementation of zigbe exist in **[Home Assistant](https://www.home-assistant.io)** (Python based open source home automation software) as part of its **[ZHA component](https://www.home-assistant.io/components/zha/)**
+**[zigpy](https://github.com/zigpy/zigpy)** is **[Zigbee protocol stack](https://en.wikipedia.org/wiki/Zigbee)** integration project to implement the **[Zigbee Home Automation](https://www.zigbee.org/)** standard as a Python 3 library. Zigbee Home Automation integration with zigpy allows you to connect one of many off-the-shelf Zigbee adapters using one of the available Zigbee radio library modules compatible with zigpy to control Zigbee based devices. There is currently support for controlling Zigbee device types such as binary sensors (e.g., motion and door sensors), sensors (e.g., temperature sensors), lightbulbs, switches, and fans. A working implementation of zigbe exist in **[Home Assistant](https://www.home-assistant.io)** (Python based open source home automation software) as part of its **[ZHA component](https://www.home-assistant.io/components/zha/)**
```

Comment:
`zigbe` -> `Zigbee`

Reply:
Corrected: zigbe -> zigpy ("A working implementation of zigpy exist in Home Assistant")

Reply:
I think Zigbee was correct. Zigpy + ZHA lets you work with Zigbee devices in HA.

Reply:
I'm sure it is meant to say "A working implementation of zigpy exist in Home Assistant as part of its ZHA component"

Reply:
Zigpy isn't implemented in Home Assistant, it exists quite independently of Home Assistant. Home Assistant's ZHA component glues together Zigpy, the radio modules, and Home Assistant, giving you a working Zigbee "stack".

Reply:
Yes exactly, and that is precisely how I read the meaning that sentence, it is semantics, just different way of saying it but the very same meaning. I guess I see the point of that sentence existence as a reference to a working implementation of zigpy, because that whole section is only there as a link to projects related to zigpy. You could indeed rewrite that sentence in many ways but keeping the same meaning, for example; 

 "A working implementation of zigpy exist in Home Assistant as part of its ZHA component".

or

 "A working implementation of zigpy exist in the ZHA component for Home Assistant".

or

 "Home Assistant's ZHA component for Home Assistant has a working implementation of zigpy."

or

 "ZHA component for Home Assistant has a working implementation of zigpy"

Regardless this is kind of a moot point to argue that here for this PR when all I really wanted to add in this PR are the PyPI links however I just noticed and changed a couple of issues I thought was spelling and grammar corrections only. Thus that sentence should not be a showstopper for this PR, instead, you could simply merge this PR for the PiPI links and then change that whole sentence to whatever you want in a separate PR, and that way you don't have to argue semantics in an unrelated PR.

Reply:
Agreed. Thanks for your help documenting these projects!